queries/html/injections.scm

```diff
@@ -25,3 +25,112 @@
   (text) @injection.content
   (#eq? @_py_config "py-config")
   (#set! injection.language "toml"))
+
+;------ Angular queries ------
+
+; {{ someBinding }}
+((text) @injection.content 
```

Comment:
With this many text node injections. i kindof feel like there'll be performance issues

Reply:
Yeah, that's way too many. We either need to add support for conditional injections, or leave them for user config.

(Alternatively -- and I'm tending more and more towards that -- we should punt all "framework" handling to an `nvim-treesitter-webdev` plugin that can be smarter about what to inject and where.)

Reply:
> (Alternatively -- and I'm tending more and more towards that -- we should punt all "framework" handling to an `nvim-treesitter-webdev` plugin that can be smarter about what to inject and where.)

`nvim-treesitter-angular` is dead.
Long live `nvim-treesitter-webdev`.

Reply:
(Joke aside, that was my first thought when I looked at nvim-treesitter-angular, which would have been a reasonable thing. Only when I learned that it doesn't do anything that nvim-treesitter wouldn't do did I press for upstreaming and archiving.)

Reply:
I do have an idea that I'll post on matrix for discussion later ~~provided I don't forget~~.